src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={tab.id === currentTab.id && 'is-active'}
```

Comment:
Would be great to create a vairable for `tab.id === currentTab.id` to avoid duplicating yourself and to increase readability

Reply:
Okay, thanks!